src/lib/config/index.ts

```diff
@@ -1,6 +1,7 @@
 import homestead from './homestead.json';
 import kovan from './kovan.json';
 import rinkeby from './rinkeby.json';
+import matic from './matic.json';
```

Comment:
Should it be called matic or polygon?

Reply:
Ack, force of habit. Polygon team seems to split the usage across both terms (using matic in more technical writing) but it makes sense to just use Polygon.